src/Components/FormCreation/CardComponents/FileQuestion.tsx

```diff
@@ -38,16 +39,51 @@ function FileQuestion({
   const numFileOptions = [1, 2, 3, 5, 10];
 
   async function handleFile(event: any) {
-    let file = event.target.files[0]; //access the file
-    //todo: check if the file extension is valid (on submission)
+    const file = event.target.files[0]; //access the file
+    //todo: check if the file extension is valid
     const formData = new FormData();
     formData.append("file", file); // appending file
-    //result will be a link to the file, which we should save to our DB
-    await FILE.fileUpload("arbitrium", file.name, formData);
+
+    //returns the url to the file in aws
+    await fileUpload("arbitrium", file.name, formData);
+    //TODO Save to DB
+  }
+
+  //download a file from AWS
+  //TODO: use for displaying download links on decision canvas
+  const [loadFile] = usePromise(
+    downloadFile,
+    {
+      bucketname: "arbitrium",
+      filename: "about.png"
+    },
+    null,
+    []
+  );
+  let link = "";
+  //Create a link to download the file :)
+  if (!loadFile.isPending) {
+    const bytes = new Uint8Array(loadFile.value.Body.data); // pass your byte response to this constructor
+    const blob = new Blob([bytes], { type: "application/octet-stream" }); // change resultByte to bytes
+    link = window.URL.createObjectURL(blob);
+
+    // const headerImg = document?.querySelector<HTMLImageElement>("#image");
+    // // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    // headerImg!.src = link;
   }
 
   return (
     <Wrapper>
+      {!loadFile.isPending ? (
+        <div>
+          <Button
+            href={link}
+            download="programs/5fab02657cf3c7788fc00d1fabout.png"
```

Comment:
Is this a temporary path? If not, would also vouch for this to be moved to an .env variable

Reply:
temporary will be replaced